.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "3rdparty/vta-hw"]
 	path = 3rdparty/vta-hw
 	url = https://github.com/apache/incubator-tvm-vta
+[submodule "3rdparty/universal"]
+	path = 3rdparty/universal
```

Comment:
I took a look at the current repo. Given that the repo contains binaries(exe and pdf) we cannot make it as a submodule (due to the requirement that all released code need to be sourc). 

We can give users instructions to go and download the library though.  It would still be nice to have the example built on top of the bfloat16 toy examples.

Reply:
See my response, we will remove Universal as a dependency! I will look in to including a feature flag to turn on the posit code paths.